libs/plutus-preprocessor/src/Debug.hs

```diff
@@ -1,17 +1,14 @@
 {-# LANGUAGE DataKinds #-}
-{-# LANGUAGE StandaloneDeriving #-}
 {-# LANGUAGE TypeApplications #-}
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 
 module Main where
 
 import Cardano.Ledger.Alonzo.Language (Language (..))
-import Cardano.Ledger.Alonzo.TxInfo (PlutusDebug (..), PlutusDebugInfo (..), debugPlutus)
+import Cardano.Ledger.Alonzo.TxInfo (debugPlutus)
 import Cardano.Ledger.BaseTypes (natVersion)
 import System.Environment (getArgs)
 import System.IO
 
-deriving instance Show PlutusDebugInfo
-
 main :: IO ()
-main = print . debugPlutus (natVersion @7) . head =<< getArgs
+main = print . debugPlutus @'PlutusV1 (natVersion @7) . head =<< getArgs -- NOTE: @'PlutusV1 does not matter since it is uses PlutusCore.Data underneath
```

Comment:
Overall the PR looks fine, but I am a bit baffled by this comment; is it true that the '@'PlutusV1' type annotation does not matter here? It looks to me that `debugPlutus` uses the version to call `evalV1` or `evalV2`. Can somebody explain why?

Reply:
Sorry @bezirg I'm at fault here. I only considered that the underlying `Data` is being considered when the compiler complained, based on the error message it was showing. But this function should indeed take into account the possibility for both versions of Plutus. I will fix this and push an update. Thanks a ton for pointing this out!

Reply:
@lehins fixed this in the latest commit. :eyes: :muscle: